extendr-api/Cargo.toml

```diff
@@ -28,7 +22,7 @@ num-complex = { version = "0.4", optional = true }
 serde = { version = "1.0", features = ["derive"], optional = true }
 
 [dev-dependencies]
-extendr-engine = { path = "../extendr-engine", version = "0.6.0" }
+extendr-engine = { path = "../extendr-engine" }
```

Comment:
Should this be workspace as well?

Reply:
In theory, yes!, in practice this is such an annoying package, that I don't want to mess with it too much already.
If multiple packages in a workspace depends on a crate, then said crate should be `workspace = true`, in my opinion.

Reply:
I think the pain point is that we have to rewrite the versions here and there, but It seems `dev-dependencies` doesn't require `version`, so I didn't include in this pull request. I don't have strong opinion here.

Reply:
I think this should be left like this. If we add another package to the workspace, that needs `extendr-engine`, I'll deal with it then.